modules/imgcodecs/test/test_tiff.cpp

```diff
@@ -147,6 +147,27 @@ TEST(Imgcodecs_Tiff, decode_infinite_rowsperstrip)
     EXPECT_EQ(0, remove(filename.c_str()));
 }
 
+TEST(Imgcodecs_Tiff, readWrite_unsigned)
+{
+    const string root = cvtest::TS::ptr()->get_data_path();
+    const string filenameInput = root + "readwrite/gray_8u.tiff";
```

Comment:
> gray_8u.tiff

vs

> gray_8u.tif

Please run tests locally before submitting.

Reply:
How to run tests locally? are there any document I can study? Thank you very much!

Reply:
8.Run testsuite locally:
1.get the required sample data by cloning https://github.com/opencv/opencv_extra (choose corresponding branch).
2.export OPENCV_TEST_DATA_PATH=<path_to_opencv_extra>/testdata.
3.execute each test binary from the build directory, e.g. ./bin/opencv_test_core.
got it！I will run it next time.